src/components/sdk/DeviceSelection/docs/VideoFilterSelection.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoFilterSelection from '../CameraSelection/VideoFilterSelection';
+
+<Meta title="SDK Components/DeviceSelection/Camera/VideoFilterSelection" />
+
+# VideoFilterSelection
+
+The `VideoFilterSelection` component renders a `Select FormField` to select a video filter. Currently, the only option is background blur.
+
+The component depends on the `DevicesProvider`. If you are using `MeetingProvider`, `DevicesProvider` is rendered by default.
+
+## Importing
+
+```javascript
+import { VideoFilterSelection } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  VideoFilterSelection
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => {
+  return (
+    <MeetingProvider>
+      <VideoFilterSelection />
```

Comment:
This needs to be wrapped by a `BackgroundBlurProvider`, else the `useBackgroundBlur` hook will throw an error.